### ISSUE TITLE
System.Net.HttpStatusCode (and potentially other "System" types)

### ISSUE THREAD

**Issue by coolymike** [Yellow-Dog-Man/Resonite-Issues#2498]
### Describe the issue.

The type System.Net.HttpStatusCode is not a data model type, causing most networking nodes to break.

Some other types like `System.DayOfWeek` seem to be affected too.

### To Reproduce

Spawn out any network related node (like GET Request) in a session with more than 1 user. This causes a world crash.

### Expected behavior

No crash

### Resonite Version Number

2024.7.7.1295

### What Platforms does this occur on?

Windows, Linux

### What headset if any do you use?

Desktop

### Log Files

```
00:46:43.078 (595 FPS)	Assembly System is not a data model assembly for this world, cannot decode type: System.Net.HttpStatusCode
00:46:43.078 (595 FPS)	Type not found: System.Net.HttpStatusCode
```

### Reporters

whippersnatchpumpkinpatch on Discord

**Comment by gameboycjp** [comment]
resrec:///U-gamethecupdog/R-FDBA6A8FDE1A8CAB24C19BF2ED2717E8535A080423129632EDA8ABFC8D8A65E6 
I used this folder in another bug report, but the first 3 objects (nearly identical) have the issue being described here on the prerelease. It only affects existing objects, these types work when pulled out with a flux tip.
![image](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/37276103/472449a2-eb9e-4cc8-8524-ffa7022a9ae9)

Here's a full log, I have the facet on my dashboard so it triggers the issue too.
[DESKTOP-3EE2DAL - 2024.7.7.1295 - 2024-07-07 17_28_57.log](https://github.com/user-attachments/files/16121706/DESKTOP-3EE2DAL.-.2024.7.7.1295.-.2024-07-07.17_28_57.log)

**Comment by coolymike** [comment]
This issue affects networked and saved items, but does not have any impact in a single player session when creating new nodes/components.

**Comment by Frooxius** [comment]
Fixed this on 2024.7.8.1130, thanks!
